newspaper/extractors.py

```diff
@@ -21,6 +21,11 @@
 from tldextract import tldextract
 from urllib.parse import urljoin, urlparse, urlunparse
 
+from extruct.jsonld import JsonLdExtractor
```

Comment:
Can we get away with this by just using the libraries newspaper already imports (e.g. lxml, BeautifulSoup) I'm not against adding extruct but it would be better to keep this library with fewer dependencies